concordia/models.py

```diff
@@ -66,6 +68,41 @@ class Collection(models.Model):
     def __str__(self):
         return self.title
 
+    def copy_images_to_collection(self, url, collection_path):
+        result2 = None
+        try:
+            result = download_async_collection.delay(url)
+            result.ready()
+            result.get()
+            result2 = check_completeness.delay()
```

Comment:
These lines (77-79) need to be deleted since check_completeness doesn't really do anything